provisioner/template_test.go

```diff
@@ -104,6 +104,16 @@ func TestManifestHashRecursiveInclude(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestSha256(t *testing.T) {
+	result, err := renderSingle(
+		t,
+		`{{ printf "%.32s" (.Values.data | sha256) }}`,
```

Comment:
sha256 is a well-defined function so I propose to implement it as is and derive shorter values if needed.